tools/Crank/run-benchmarks.ps1

```diff
@@ -77,19 +74,16 @@ $homePath = if ($isLinuxApp) { "/home/$UserName/FunctionApps/$FunctionApp" } els
 $functionAppPath = if ($isLinuxApp) { "/home/$UserName/FunctionApps/$FunctionApp/site/wwwroot" } else { "C:\FunctionApps\$FunctionApp\site\wwwroot" }
 $tmpLogPath = if ($isLinuxApp) { "/tmp/functions/log" } else { 'C:\Temp\Functions\Log' }
 
-if ($UseHttps) {
-    $aspNetUrls = "http://localhost:5000;https://localhost:5001"
-    $profileName = "localHttps"
-}
-else {
-    $aspNetUrls = "http://localhost:5000"
-    $profileName = "local"
-}
+ $aspNetUrls = "http://localhost:5000"
+ $profileName = "default"
 
 $crankArgs =
     '--config', $crankConfigPath,
     '--scenario', $Scenario,
     '--profile', $profileName,
+    '--chart',
```

Comment:
These chart/collectCounters options render a chart showing all sorts of useful application details